openshift/default-catalog-consistency/pkg/check/catalog.go

```diff
@@ -0,0 +1,122 @@
+package check
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+)
+
+// AllCatalogChecks returns a list of all catalog checks to be performed.
+func AllCatalogChecks() []CatalogCheck {
+	return []CatalogCheck{
+		CatalogHasNotBundleObject(),
+		CatalogHasValidMetadataForChannelHeads(),
+	}
+}
+
+// CatalogHasNotBundleObject verifies that any bundle has not olm.bundle.object.
```

Comment:
I'm ussing the suggest changes button for some reason =S

```
// CatalogHasNoBundleObjectProperties verifies that no bundle has declares a olm.bundle.object property.
```

Reply:
Accepted 👍